app/services/pul_dspace_connector.rb

```diff
@@ -87,9 +87,14 @@ def download_file(url, filename)
       end
     end
 
+    # rubocop:disable Metrics/MethodLength
     def checksum_file(filename, bitstream)
       checksum_class = Digest.const_get(bitstream["checkSum"]["checkSumAlgorithm"])
-      if checksum_class.file(filename).hexdigest != bitstream["checkSum"]["value"]
+      checksum = checksum_class.file(filename)
+      hexdigest = checksum.hexdigest
+      base64 = checksum.base64digest
+      bitstream["checkSum"]["base64"] = base64
+      if hexdigest != bitstream["checkSum"]["value"]
```

Comment:
Just double-checking that this was indeed meant to be `value` and not `base64`

Reply:
@hectorcorrea Yes, that's correct as-is. DSpace needs the hexdigest and S3 needs the base64digest.